.github/workflows/pull_requests.yml

```diff
@@ -28,6 +28,7 @@ jobs:
           flag-name: run-jvm-${{ join(matrix.*, '-') }}
           parallel: true
           base-path: src/main/java
+          fail-on-error: ${{ matrix.os != 'macos' }}
```

Comment:
https://github.com/coverallsapp/homebrew-coveralls/issues/65#issuecomment-3259869882 following this advice before coveralls starts working with macos 15.5